data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/DynamoDBService.java

```diff
@@ -129,15 +129,15 @@ public void shutdown() {
      * Future optimization can be done to accept configuration changes
      */
     public void init() {
-        LOG.info("Start initialize DynamoDB service");
+        LOG.info("Try to initialize DynamoDB service");
```

Comment:
A lot of these logs really do belong in DEBUG level. Can you make them DEBUG in a future PR (once more thorough testing is done with the verbose logs)?

Reply:
Happy to do it.